src/extension.ts

```diff
@@ -20,13 +20,17 @@ interface Rangeable {
 }
 
 export function activate(context: ExtensionContext) {
-    const eventDisposable = (workspace as any).onWillSaveTextDocument(e => {
+    const eventDisposable = workspace.onWillSaveTextDocument(e => {
```

Comment:
This will generate compilation error

```
src/extension.ts(23,39): error TS2339: Property 'onWillSaveTextDocument' does not exist on type 'typeof workspace'.
```